maven/src/main/java/org/owasp/dependencycheck/maven/BaseDependencyCheckMojo.java

```diff
@@ -1586,6 +1586,7 @@ private ExceptionCollection collectDependencies(Engine engine, MavenProject proj
                 mixedLangSet.addInclude("npm-shrinkwrap.json");
                 mixedLangSet.addInclude("Gopkg.lock");
                 mixedLangSet.addInclude("go.mod");
+                mixedLangSet.addInclude("yarn.lock");
```

Comment:
We should also include the pnpm-lock file here.

```suggestion
                mixedLangSet.addInclude("yarn.lock");
                mixedLangSet.addInclude("pnpm-lock.yaml");
```